apps/krew-website/src/pages/index.astro

```diff
@@ -1,41 +1,263 @@
+            <span class="text-gray-500">Module bundling</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">Webpack 5</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Service Worker</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">Yes</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Performance patterns</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">-</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Server-side rendering</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">No</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">API</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">NGRX component store</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Hosting</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">-</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Other details</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">Tailwind CSS</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Author</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">Aalt / Guido</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Maintainers</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500 text-right">Aalt / Eline / Guido</span>
+          </div>
+          <div class="flex border-t border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Progress</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500">85%</span>
+          </div>
+          <div class="flex border-t border-b mb-6 border-gray-200 py-2 dark:border-gray-800">
+            <span class="text-gray-500">Version</span>
+            <span class="ml-auto text-gray-900 dark:text-gray-500">0.6</span>
+          </div>
+
+          <div class="flex">
+            <a href='https://krew-angular.test.m4m.io/' target='_blank' class="transition flex text-white mr-3 bg-indigo-500 border-0 py-2 px-6 focus:outline-none hover:bg-indigo-600 rounded">Launch</a>
```

Comment:
I think this could be done together when the html here is extracted to loop over a collection to display the details about each project, instead of repeating the same html.

Reply:
I think that can be done at the same time as when this section is being made dynamically.
So having the sections coming from a collection and loop over it, instead of defining the same html multiple times.